aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,427 @@
-More content coming soon.
+Messaging within HPE applications should be informative, concise, and trustworthy.
+It should be delivered in active, positive, objective, conversational
+American English and written from a third-person [point of view](#point-of-view).
+Executed properly, messaging within HPE applications should leave users
+feeling and perceiving HPE and its products to be intelligent, experienced,
+inclusive, and non-threatening.
+
+## Messaging Voice, Tone, and Content
```

Comment:
It might be a good idea to change the title of the page to something broader like "Communication" or "Content" (instead of "Voice and Tone"). If we do that, then the heading of this particular section can be updated to "Voice and Tone"

Reply:
I am going to leave this suggestion for further discussion after this PR.